### PR TITLE
fixes #3724 - correcting Red Hat names in some UI texts

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: app/views/dashboard/welcome.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <p>
     <%= _('You may operate Foreman in basic mode, in which it acts as a reporting and external node classifier or you may also turn on unattended mode operation in which Foreman creates and manages the configuration files necessary to completely configure a new host.') %>
 
-    <%= _('When operating in unattended mode Foreman will require more information, so expect more questions, but it will be able to automate host installations for redhat, debian, suse and solaris operating systems (and their clones), see') %>
+    <%= _('When operating in unattended mode Foreman will require more information, so expect more questions, but it will be able to automate host installations for Red Hat, Debian, Suse and Solaris operating systems (and their clones), see') %>
```
Comment (domcleal) [inline app/views/dashboard/welcome.html.erb]: SUSE?

Comment (lzap) [inline app/views/dashboard/welcome.html.erb]: Sure, we do support both Red Hat systems and Suse systems. I know this
is OpenSuse and SLES, but do we really name it all here?

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

Comment (lzap) [inline app/views/dashboard/welcome.html.erb]: Okay I see I had bad case there, correcting :-)

**Comment by domcleal** [comment]
Thanks @lzap, merged as cb4177d for Foreman 1.4.0.

**Pull request closed without merging**